### PR TITLE
Fixed #90

### LINKED ISSUE #90: npm install fails due to malicious package deprecation by NPM
**Describe the bug**

Cannot `npm install` because `flatmap-stream` (a dependency of `vscode`) was flagged as a **malicious package** and was deprecated by NPM.

I’m guessing that’s why bot package updates are failing.

**To Reproduce**

Fork repo, git clone repo and `npm install`.

Upgrading dependency `vscode` to `^1.1.26` fixes the vulnerability and `npm install` works, but unit tests fail and the learning curve is a bit step for me to contribute a fix.

### PR DESCRIPTION
**Pull request by sunknudsen**
Upgraded vscode dependency to 1.1.26 to fix #90. Tests failed (not sure if this is due to upgrade). Fixed tests and suggested new icon and theme to make extension stand out on extension marketplace.

### CODE REVIEW

**Comment by sunknudsen** [comment]
This PR should also fix pending Greenkeeper issues.

File: package.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
   ],
   "icon": "images/icon.png",
   "galleryBanner": {
-    "color": "#5c2d91",
```
Comment (sleistner) [inline package.json]: Do you meant to change the banner color?

**Comment by sleistner** [comment]
Thanks for the efforts, nice PR!

File: package.json (hunk 1 above)
Comment (sunknudsen) [reply package.json]: Yes. This commit is suggesting a new icon and banner color. If you feel the design needs more work (it was put together quickly), feel free to discard it and let me know if you would like to bounce other ideas.

**Pull request merged**